virtcontainers/container.go

```diff
@@ -1181,6 +1181,19 @@ func (c *Container) addResources() error {
 		return c.sandbox.agent.onlineCPUMem(vcpusAdded)
 	}
 
+	// Container is being created, try to add the memory specified
+	mem := c.config.Resources.Mem
+	if mem != 0 {
+		virtLog.Debugf("hot adding %d B memory", mem)
+		sizeMB := int(mem / 1024 / 1024)
+		// sizeMB needs to be divisible by 2
+		sizeMB = sizeMB + sizeMB%2
```

Comment:
It might be better to move the log call to here and log `sizeMB` (and maybe `mem` too):

```
virtLog.WithField("memory-mb", sizeMB).Info("hot adding memory")
```

Same comment for the log call in `removeResources()`.